src/h264/H264Packetizer.h

```diff
@@ -16,6 +18,11 @@ class H264Packetizer : public H26xPacketizer
 	H264Packetizer();
 	std::unique_ptr<MediaFrame> ProcessAU(BufferReader &reader) override;
 
+	void SetScteData(Buffer data);
```

Comment:
this is not really a setter, use a single method:
```suggestion
	void AppendScteData(uint64_t ts, const Buffer& data, uint8_t repeatCount);
```

Reply:
ok